src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.FigureSupplier;
+
+public class Main {
+    private static final int FIGURES_NUMBER = 6;
+
+    public static void main(String[] args) {
+
+        Figure[] figures = new Figure[FIGURES_NUMBER];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        for (int i = 0; i < figures.length; i++) {
+
```

Comment:
```suggestion
        Figure[] figures = new Figure[FIGURES_NUMBER];
        FigureSupplier figureSupplier = new FigureSupplier();
        for (int i = 0; i < figures.length; i++) {
```

lets remove redundant empty lines from solution